db/index.js

```diff
@@ -0,0 +1,54 @@
+var videos = require('./videoData.js')
+
+// console.log(videos)
+
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/media', { useNewUrlParser: true, useUnifiedTopology: true });
+
+var db = mongoose.connection;
+
+db.on('error', function (err) {
+  console.log('mongoose connection error: ', err);
+});
+
+db.once('open', function () {
+  console.log('mongoose connected successfully');
+});
+
+var mediaSchema = mongoose.Schema({
+  id: {type: Number, unqiue: true},
+  mediaType: String,
+  url: String
+})
+
+var Item = mongoose.model('Item', mediaSchema);
+
+var insertVideos = (arr) => {
+  for (let i = 0; i < 20; i++) {
+    var videoId = arr.videos[i].id.videoId
+    var newItem = new Item({
+      id: i,
+      mediaType: 'video',
+      url: `https://www.youtube.com/embed/${videoId}`
+    })
+    newItem.save()
+  }
+}
+
+var insertPictures = () => {
+  for (let i = 20; i < 100; i++) {
+    var newItem = new Item({
+      id: i,
+      mediaType: 'photo',
+      url: `https://loremflickr.com/320/240?lock=${i}`
```

Comment:
These seem to be API-hosted placeholder images. Since this is a media-centric module, you might consider taking a look at [the line on GLearn - FEC Project Milestones](https://learn-2.galvanize.com/cohorts/1677/blocks/94/content_files/Front%20End%20Capstone/exercises/project_milestones.md) which talks about transferring and hosting photos from through your own cloud account on Amazon or S3, and ask a TM to clarify. (I used [Cloudinary](https://cloudinary.com/), which doesn't require a credit card.)

Reply:
Thanks for putting Cloudinary on my radar! I just made that change.